lnd_test.go

```diff
@@ -5134,23 +5115,23 @@ func testInvoiceSubscriptions(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("not all invoices settled")
 	}
 
-	ctxt, _ = context.WithTimeout(ctxb, timeout)
+	ctxt, _ = context.WithTimeout(ctxb, channelCloseTimeout)
 	closeChannelAndAssert(ctxt, t, net, net.Alice, chanPoint, false)
 }
 
 // testBasicChannelCreation test multiple channel opening and closing.
 func testBasicChannelCreation(net *lntest.NetworkHarness, t *harnessTest) {
 	const (
 		numChannels = 2
-		timeout     = time.Duration(time.Second * 5)
 		amount      = maxBtcFundingAmount
 	)
 
 	// Open the channel between Alice and Bob, asserting that the
 	// channel has been properly open on-chain.
 	chanPoints := make([]*lnrpc.ChannelPoint, numChannels)
 	for i := 0; i < numChannels; i++ {
-		ctx, _ := context.WithTimeout(context.Background(), timeout)
+		ctx, _ := context.WithTimeout(context.Background(),
```

Comment:
Nit: we can just define `context.Background()` as `ctxb` as we usually do in the other tests.

Reply:
Yep, several places this can be done. Leaving as is for now, as PR is big enough.